src/components/WalletDropdown/DefaultMenu.tsx

```diff
@@ -0,0 +1,258 @@
+import { Trans } from '@lingui/macro'
```

Comment:
Wallet went through a number of revisions, code bases, styling, etc....

I tried to use all uni stuff since this is a uni component.  Please leave as much feedback to simplify and improve to match uni standards so I can do better in future. thanks